src/CommunityToolkit.Maui/Behaviors/MaxLengthReachedBehavior.shared.cs

```diff
@@ -50,9 +50,10 @@ public bool ShouldDismissKeyboardAutomatically
 	}
 
 	/// <inheritdoc/>
-	protected override void OnViewPropertyChanged(object? sender, PropertyChangedEventArgs e)
+	protected override void OnViewPropertyChanged(InputView sender, PropertyChangedEventArgs e)
```

Comment:
Maybe we should use interface here

Reply:
Since we are on `Community.Maui` projects it's safe to use the `Maui.Controls` objects. Actually could be safer than using the interface... Because here we force the dev to really use an InputView and we designed this behavior to work on this control. An interface will be more "generic".

Reply:
For Behaviors, we can't use interfaces because `public IList<Behavior> Behaviors` isn't defined in any interface; it is only defined in `VisualElement`:

https://github.com/dotnet/maui/blob/ec33802c4ab453e13a24b1fc63bb019cd28152f9/src/Controls/src/Core/VisualElement.cs#L421-L424

This means that for `BaseBehavior<TView>`, the lowest that `TView` can be constrained is to `VisualElement`:
```cs
public abstract class BaseBehavior<TView> : Behavior<TView> where TView : VisualElement
```